### ISSUE TITLE
Newpipe Sponsorblock no longer playing Youtube videos

### ISSUE THREAD

**Issue by simula67** [polymorphicshade/NewPipe#130]
### Checklist

- [x] I am using the latest version - x.xx.x 
- [ ] This issue CANNOT be reproduced with the same version of upstream NewPipe 
- [x] I checked, but didn't find any duplicates (open OR closed) of this issue in the repo. 
- [x] I have read the contribution guidelines given at https://github.com/TeamNewPipe/NewPipe/blob/HEAD/.github/CONTRIBUTING.md.
- [x] This issue contains only one bug. I will open one issue for every bug report I want to file.

### Steps to reproduce the bug

Open any video on Youtube

### Actual behavior

Newpipe sponsorblock gives an error: "Sorry, something went wrong"

### Expected behavior

Video should play normally.

### Screenshots/Screen recordings

Logs shown below

## Exception
* __User Action:__ requested stream
* __Request:__ https://www.youtube.com/watch?v=u5CguqywlBk
* __Content Country:__ GB
* __Content Language:__ en-GB
* __App Language:__ en_GB
* __Service:__ YouTube
* __Version:__ 0.21.15
* __OS:__ Linux Android 11 - 30
<details><summary><b>Crash log </b></summary><p>

```
org.schabi.newpipe.extractor.stream.StreamInfo$StreamExtractException: Could not get any stream. See error variable to get further details.
	at org.schabi.newpipe.extractor.stream.StreamInfo.extractStreams(StreamInfo.java:194)
	at org.schabi.newpipe.extractor.stream.StreamInfo.getInfo(StreamInfo.java:72)
	at org.schabi.newpipe.extractor.stream.StreamInfo.getInfo(StreamInfo.java:64)
	at org.schabi.newpipe.util.ExtractorHelper.lambda$getStreamInfo$3(ExtractorHelper.java:116)
	at org.schabi.newpipe.util.ExtractorHelper.$r8$lambda$YTHJjScxCJNO1LTCqs3IKy35iyY(Unknown Source:0)
	at org.schabi.newpipe.util.ExtractorHelper$$ExternalSyntheticLambda6.call(Unknown Source:4)
	at io.reactivex.rxjava3.internal.operators.single.SingleFromCallable.subscribeActual(SingleFromCallable.java:43)
	at io.reactivex.rxjava3.core.Single.subscribe(Single.java:4813)
	at io.reactivex.rxjava3.internal.operators.single.SingleDoOnSuccess.subscribeActual(SingleDoOnSuccess.java:35)
	at io.reactivex.rxjava3.core.Single.subscribe(Single.java:4813)
	at io.reactivex.rxjava3.internal.operators.maybe.MaybeFromSingle.subscribeActual(MaybeFromSingle.java:41)
	at io.reactivex.rxjava3.core.Maybe.subscribe(Maybe.java:5330)
	at io.reactivex.rxjava3.internal.operators.maybe.MaybeConcatArray$ConcatMaybeObserver.drain(MaybeConcatArray.java:153)
	at io.reactivex.rxjava3.internal.operators.maybe.MaybeConcatArray$ConcatMaybeObserver.request(MaybeConcatArray.java:78)
	at io.reactivex.rxjava3.internal.operators.flowable.FlowableElementAtMaybe$ElementAtSubscriber.onSubscribe(FlowableElementAtMaybe.java:66)
	at io.reactivex.rxjava3.internal.operators.maybe.MaybeConcatArray.subscribeActual(MaybeConcatArray.java:42)
	at io.reactivex.rxjava3.core.Flowable.subscribe(Flowable.java:15753)
	at io.reactivex.rxjava3.internal.operators.flowable.FlowableElementAtMaybe.subscribeActual(FlowableElementAtMaybe.java:36)
	at io.reactivex.rxjava3.core.Maybe.subscribe(Maybe.java:5330)
	at io.reactivex.rxjava3.internal.operators.maybe.MaybeToSingle.subscribeActual(MaybeToSingle.java:46)
	at io.reactivex.rxjava3.core.Single.subscribe(Single.java:4813)
	at io.reactivex.rxjava3.internal.operators.single.SingleSubscribeOn$SubscribeOnObserver.run(SingleSubscribeOn.java:89)
	at io.reactivex.rxjava3.core.Scheduler$DisposeTask.run(Scheduler.java:614)
	at io.reactivex.rxjava3.internal.schedulers.ScheduledRunnable.run(ScheduledRunnable.java:65)
	at io.reactivex.rxjava3.internal.schedulers.ScheduledRunnable.call(ScheduledRunnable.java:56)
	at java.util.concurrent.FutureTask.run(FutureTask.java:266)
	at java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.run(ScheduledThreadPoolExecutor.java:301)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1167)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:641)
	at java.lang.Thread.run(Thread.java:923)

```
</details>
<hr>

### Device info

 - Android version/Custom ROM version: Android: 11, OxygenOS: 11.1.2.2
 - Device model: Oneplus 6 (Oneplus A6000)

### Remarks

This seems to be fixed upstream. An update from upstream and a new release might be sufficient to fix the problem.

**Comment by PacoBell** [comment]
"This issue CANNOT be reproduced with the same version of upstream NewPipe"

That's not true, is it? You even said it was fixed upstream.

**Comment by simula67** [comment]
Yes, it's true. They fixed it a few hours ago: https://github.com/TeamNewPipe/NewPipe/releases

**Comment by PacoBell** [comment]
I was referring specifically to the fact that this issue you reported here to this issue tracker was, in fact, reproducible upstream with the same version. Hence, your bug report is both invalid and unhelpful for this repository.

**Comment by simula67** [comment]
Yes, sorry. It is not reproducible with the same version. It is however, fixed in the newer version.

**Comment by PacoBell** [comment]
C'mon, man. They pinned the issue reproducing the exact same thing upstream here. 

https://github.com/TeamNewPipe/NewPipe/issues/7734

**Comment by i13e** [comment]
#128 should address this

**Comment by polymorphicshade** [comment]
This has been fixed in version 0.21.16.

**Comment by simula67** [comment]
Thanks, just got the new update. This problem is fixed 👍
